apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/V2Server.java

```diff
@@ -94,7 +94,8 @@ protected void initialize() throws ServletException {
      * global history.
      */
     List<Object> plainProviders = new ArrayList<>();
-    registerProvider(plainProviders);
```

Comment:
This change was breaking V2, which broke deployments. I reverted back the original and deprecated method.